src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static WorkWithFile workWithFile = new WorkWithFile();
+    private static ReadData readData = new ReadData();
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
+            writer.write(workWithFile.createReport(readData.readFile(fromFileName)));
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t write to file");
+        }
```

Comment:
No, no, no) The method getStatistic must call your methods, but the logic for writing must be placed in a separate private method
```suggestion
        writeData(toFileName, workWithFile.createReport(readData.readFile(fromFileName));
```